etc/default/zfs.in

```diff
@@ -9,6 +9,12 @@
 # To enable a boolean setting, set it to yes, on, true, or 1.
```

Comment:
Since you're touching this file, please replace the ZoL on L1 with OpenZFS. I held off on it in 1966e959cab25d6e535eeb88e0b2ad9590881eb7 for this exact scenario